civcraftConfigs/betconfig.yml

```diff
@@ -156,4 +156,104 @@ monster:
      type: ENDERMAN
      spawn_chance: 0.0111
      identifier: enderman
-     maximum_light_level: 7
\ No newline at end of file
+     maximum_light_level: 7
+  Iceplains:
+  updatetime: 1m
+  areas:
+    biomes:
+     - ROOFED_FOREST
+     - MEGA_SPRUCE_TAIGA
+     - ICE_PLAINS
+  mobconfig:
+    OPirongolem:
+     identifier: PowerGolem
+     type: IRON_GOLEM
+     name: Power Golem
+     on_hit_message: Git Rekt Son
+     spawn_chance: 0.1
+     drops:
+      DasRekker:
+       material: WOOD_SWORD 
+       amount: 1
+       chance: 0.01
+       lore: Das Rekker Wood Sword
+       enchants:
+        FA2:
+         enchant: FIRE_ASPECT
+         level: 2
+         S2:
+         enchant: DAMAGE_ALL
+         level: 2
+     buffs:
+      s4:
+       type: INCREASE_DAMAGE
+       level: 4
+      Fast4:
+       type: SPEED
+       level: 4
+     on_hit_debuffs:
+      Confused:
+       type: CONFUSION
+       level: 1
+       duration: 10s
+       chance: 1.0 
+      blinded:
+       type: BLINDNESS
+       level: 1
+       duration: 5s
+       chance: 1.0 
+      Slowness:
+       type: SLOW
+       level: 2
+       duration: 5s
+       chance: 1.0 
+     blocks_to_spawn_on:
+      - STONE
+      - DIRT
+      - GRASS
+      - STATIONARY_LAVA
+      - STATIONARY_WATER 
+     blocks_to_spawn_in:
+      - AIR
+      - STATIONARY_LAVA
+      - STATIONARY_WATER 
+     health: 3000
+     despawn_on_chunk_unload: true
+     can_pickup_items: false
+     y_spawn_range: 16
+
+spawner:
```

Comment:
Oh, so a spawner has to be created. Hmm, I thought I did that with the config, but it seems some other way had to be done instead. Nevertheless, having a spawner in the griefed and dead city of NSR could contribute to the lore of Bet. Maybe we can do that another day.

I'll cut out the code on that section until there's a way to make mobs spawn within a radius of a location instead of just biome dependent.